src/views/index.js

```diff
@@ -0,0 +1,124 @@
+const loadMainContent = () => {
+  const mainDiv = document.getElementById('app');
+
+  let initialContent = `<div class="container-fluid w-100 m-0 p-0 mb-3">
+    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+      <span class="navbar-brand h1">Amazing TodoList</span>
+    </nav>
+  </div>`;
+
+  initialContent += `<div class="container pt-3">
+    <div class="row">
+      <div class="col-md-4 col-sm-12 mt-3" id="projects-div">
+        <div class="input-group mb-3" id="project-container">
+          <input
+            type="text"
+            id="new-project-input"
+            class="form-control"
+            placeholder="New project name"
+            aria-label="Project name"
+            aria-describedby="basic-addon1"
+            required=""
+          />
+          <button type="button" class="btn btn-primary" id="btn-add-project">Create</button>
+        </div>
+        <!-- Projects View renders the anchors <a> -->
+
+      </div>
+      <div class="col-md-8 col-sm-12 d-flex-column mt-3">
+        <div class="row justify-content-center mb-3">
+          <!-- Button trigger modal -->
+          <button
+          id="trigger-modal"
+            type="button"
+            class="btn btn-primary"
+            data-toggle="modal"
+            data-target="#exampleModal"
+          >
+            Create todo for Default
+          </button>
+          <!-- Modal -->
+          <div
+            class="modal fade"
+            id="exampleModal"
+            tabindex="-1"
+            role="dialog"
+            aria-labelledby="exampleModalLabel"
+            aria-hidden="true"
+          >
+            <div class="modal-dialog" role="document">
+              <div class="modal-content">
+                <div class="modal-header">
+                  <h5 class="modal-title" id="exampleModalLabel">New Todo</h5>
+                  <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">×</span>
+                  </button>
+                </div>
+                <div class="modal-body">
+                  <form id="create-todo-form">
+                    <div class="form-group">
+                      <label>Name</label>
+                      <input
+                        type="text"
+                        class="form-control"
+                        id="exampleInputEmail1"
+                        aria-describedby="emailHelp"
```

Comment:
User shouldn't be able to create empty todo right?
You can apply `required` to all the input here
screenshot from your app below
![image](https://user-images.githubusercontent.com/31897434/78202200-8c385680-748b-11ea-89e9-0cec37edba7b.png)

Reply:
Validation for create and edit form of todos